tensorboard/webapp/widgets/line_chart_v2/lib/composite_layout.ts

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {RectLayout, LayoutStrategy, LayoutOption} from './layout';
+import {Rect} from './types';
+
+export class CompositeLayout extends RectLayout {
+  private readonly superposedLayouts: RectLayout[];
+
+  constructor(config: LayoutOption, contentColumns: RectLayout[]) {
```

Comment:
A `RectLayout` is also a composite layout, since it takes a rectangular
array of child `RectLayout`s. In what way is a `CompositeLayout`
different? Why does it only take columns?